components/n-ui/tracking/ft/index.js

```diff
@@ -126,27 +126,14 @@ const oTrackingWrapper = {
 				pageViewConf.context['teaser_tests'] = abTestHelpers.getTeaserTestContext(document);
 			}
 
-			// barriers
-			let barrierType = document.querySelector('[data-barrier]');
-
-			if (barrierType) {
-				pageViewConf.context.barrier = true;
-				pageViewConf.context.barrierType = barrierType.getAttribute('data-barrier');
-			};
-
 			// FIXME - should not fire on barriers, but needs to be around for a while data analytics fix their SQL
 			// Page view must not be triggered in any form of frameset, only a genuine page view, or the error page domain, as error pages are served in iframes.
 			if (window === window.top || window.location.hostname === 'errors-next.ft.com') {
 				oTracking.page(pageViewConf.context);
 			}
 
-			if (barrierType) {
-				const customTracking = JSON.parse(barrierType.getAttribute('data-tracking-object'));
-
-				broadcast('oTracking.event', Object.assign({
-					category: 'barrier',
-					action: 'view'
-				}, context, customTracking));
+			if (window.FT && window.FT.nUi) {
+				window.FT.nUi.trackingContext = context;
```

Comment:
I'm guessing this is being set so this data can be picked up from the outside... what will pick it up?

Reply:
https://github.com/Financial-Times/next-product/pull/752/files#diff-096103d8490b885f9d5037baea6aea80R82 and https://github.com/Financial-Times/next-epaper/pull/35/files#diff-a93ac81418745fbaa6d1e0e4a4171132R20

Reply:
Could you add a comment to the code with these references? It's scary setting a global and not knowing who or what consumes it!

Reply:
I've added a more generic comment... direct references tend to get outdated and are unlikely to get updated when this gets used elsewhere. At least there's some indication and a "why" now... hopefully that will suffice?

Reply:
Thanks!